### PR TITLE
Release in ES2015 format

### PR DESCRIPTION
**Pull request by timmathews**
I think this is good to go. I updated package.json to publish to @signalk on npm, so I suspect that the packages that use signalk-client will need to be updated to use it at that path.

It is published and currently `npm -i signalk-client` installs the pre-build-script version, `npm -i @signalk/signalk-client` installs this version.

Babel compiles to es5 and babili minifies. Seemed excessive to pull in WebPack just for minification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,56 @@
...
 
-Usage:
-=====
+## Usage
 
-Add:
 ```
-var signalkClient = require('signalk-client').Client;
-var signalk = new signalkClient;
+var SignalKClient = require('signalk-client').Client;
```
Comment (tkurki) [inline README.md]: Should be @signalk/signalk-client, right?

**Comment by tkurki** [comment]
Other than the single comment I'm good for merging & updating stuff that uses this. If something is not yet using Greenkeeper I think we should install that to, very straightforward to get notifications of the updates, all you need to do then is npm publish.

**Comment by tkurki** [comment]
Could you rebase this on master? Let's get this finished!

**Comment by timmathews** [comment]
Should we bump the version to 0.1.0?

**Comment by tkurki** [comment]
Sure. And go for @signalk and drop the _signalk-_ prefix from the package name.

**Pull request merged**